### PR TITLE
Android new login

### PR DESCRIPTION
**Pull request by gcalero**
Implement the following parts of the new design proposal https://highfidelity.atlassian.net/wiki/spaces/Android/pages/292225108/New+Design+Proposal

+ Login
+ Profile picture + user name in drawer menu
+ Privacy Policy
+ Haptic feedback on back, view and mic buttons
+ Refactor using android fragments

Test plan (Android only):
- On startup, you should see the Hifi log instead of the grey screen.
- In the hamburger menu, you should see a Hifi logo next to "Login".  Logging in should replace the logo with your profile picture (set here: https://highfidelity.com/profile/edit) and "Login" with your display name.  Logging out should return to the Hifi logo and "Login"
- Clicking "Privacy Policy" in the hamburger menu should bring up a scrollable wall of text.
- Logout and go to a domain where you need to login (you may need to set one up).  When you click on the domain, you should be brought to the "Login" Android Activity.
- Go to any domain.  The arrow in the top left and the two buttons in the top right (mode and audio) should trigger a short haptic pulse when you click on them.
- Test on multiple devices.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8903/app_12935_PR_120de92c9e284bf5b792bc8d85845536583204f8.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8911/HighFidelity-Beta-PR12935-8f34c02ab27ce994da416db606014d3c7f061f74.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8911/HighFidelity-Beta-Interface-PR12935-8f34c02ab27ce994da416db606014d3c7f061f74.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8911/HighFidelity-Beta-PR12935-8f34c02ab27ce994da416db606014d3c7f061f74.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8911/HighFidelity-Beta-Interface-PR12935-8f34c02ab27ce994da416db606014d3c7f061f74.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9088/app_12935_PR_5284c4edae1ab8b37881be4127b2f63cc93a923f.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9090/app_12935_PR_51f9b763ff6545e3c9ef7e9cf1489b183fbecaee.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9098/app_12935_PR_55a946c6c7bfc35903c3ad5425794e69e94b6811.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9098/HighFidelity-Beta-PR12935-55a946c6c7bfc35903c3ad5425794e69e94b6811.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9098/HighFidelity-Beta-Interface-PR12935-55a946c6c7bfc35903c3ad5425794e69e94b6811.dmg

**Comment by gcalero** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9120/app_12935_PR_d33855b860d702a002027c5684c79fc6ee2fd719.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9120/HighFidelity-Beta-PR12935-d33855b860d702a002027c5684c79fc6ee2fd719.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9120/HighFidelity-Beta-Interface-PR12935-d33855b860d702a002027c5684c79fc6ee2fd719.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9120/HighFidelity-Beta-PR12935-d33855b860d702a002027c5684c79fc6ee2fd719.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9120/HighFidelity-Beta-Interface-PR12935-d33855b860d702a002027c5684c79fc6ee2fd719.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9178/app_12935_PR_2673a06d6f083f5899df347742e39e3963193a12.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9178/HighFidelity-Beta-PR12935-2673a06d6f083f5899df347742e39e3963193a12.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9178/HighFidelity-Beta-Interface-PR12935-2673a06d6f083f5899df347742e39e3963193a12.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9178/HighFidelity-Beta-PR12935-2673a06d6f083f5899df347742e39e3963193a12.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9178/HighFidelity-Beta-Interface-PR12935-2673a06d6f083f5899df347742e39e3963193a12.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9218/app_12935_PR_f9fc549e7d50893dd37f7c37386f608f6ef4c3c7.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9218/HighFidelity-Beta-PR12935-f9fc549e7d50893dd37f7c37386f608f6ef4c3c7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9218/HighFidelity-Beta-Interface-PR12935-f9fc549e7d50893dd37f7c37386f608f6ef4c3c7.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9257/app_12935_PR_0d08bc6e7c19cd9e276c7c34af5cc5caed6d7f29.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9261/app_12935_PR_c27f415fea2e28979cba0204940979049e78157a.apk

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9261/HighFidelity-Beta-Interface-PR12935-c27f415fea2e28979cba0204940979049e78157a.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9261/HighFidelity-Beta-PR12935-c27f415fea2e28979cba0204940979049e78157a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9261/HighFidelity-Beta-Interface-PR12935-c27f415fea2e28979cba0204940979049e78157a.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9302/app_12935_PR_ce8bd1368510fc73348ffa8e9745731bd7a6a97a.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9302/HighFidelity-Beta-PR12935-ce8bd1368510fc73348ffa8e9745731bd7a6a97a.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9302/HighFidelity-Beta-Interface-PR12935-ce8bd1368510fc73348ffa8e9745731bd7a6a97a.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9302/HighFidelity-Beta-PR12935-ce8bd1368510fc73348ffa8e9745731bd7a6a97a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9302/HighFidelity-Beta-Interface-PR12935-ce8bd1368510fc73348ffa8e9745731bd7a6a97a.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9408/HighFidelity-Beta-PR12935-3a1bfefc22444b628db4d54c604527c8f02c11bd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9408/HighFidelity-Beta-Interface-PR12935-3a1bfefc22444b628db4d54c604527c8f02c11bd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9408/HighFidelity-Beta-PR12935-3a1bfefc22444b628db4d54c604527c8f02c11bd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9408/HighFidelity-Beta-Interface-PR12935-3a1bfefc22444b628db4d54c604527c8f02c11bd.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9433/app_12935_PR_d0ab81d1f9dfa8b0c26f3b6d9920d2ef3bae70c6.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9437/app_12935_PR_c97e751e48ed1674cbed0f1b95b5de191c155e49.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9448/app_12935_PR_ec838b141781ba1f4c5fde97d7272c2d6d9e7819.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9448/HighFidelity-Beta-PR12935-ec838b141781ba1f4c5fde97d7272c2d6d9e7819.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9448/HighFidelity-Beta-Interface-PR12935-ec838b141781ba1f4c5fde97d7272c2d6d9e7819.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9475/app_12935_PR_a767540b76f59b9fd5dc76de68a953f07fcd094f.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9475/HighFidelity-Beta-PR12935-a767540b76f59b9fd5dc76de68a953f07fcd094f.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9475/HighFidelity-Beta-Interface-PR12935-a767540b76f59b9fd5dc76de68a953f07fcd094f.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9514/app_12935_PR_868b58e66d9f8232c53c1ede78a5ae5fdf9be7eb.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9514/HighFidelity-Beta-PR12935-868b58e66d9f8232c53c1ede78a5ae5fdf9be7eb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9514/HighFidelity-Beta-Interface-PR12935-868b58e66d9f8232c53c1ede78a5ae5fdf9be7eb.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9514/HighFidelity-Beta-PR12935-868b58e66d9f8232c53c1ede78a5ae5fdf9be7eb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9514/HighFidelity-Beta-Interface-PR12935-868b58e66d9f8232c53c1ede78a5ae5fdf9be7eb.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9523/app_12935_PR_087c6ff730afe126fb19e09da05b6d312db27972.apk

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9523/HighFidelity-Beta-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9523/HighFidelity-Beta-Interface-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.exe

**Comment by gcalero** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9562/app_12935_PR_087c6ff730afe126fb19e09da05b6d312db27972.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9562/HighFidelity-Beta-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9562/HighFidelity-Beta-Interface-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9562/HighFidelity-Beta-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9562/HighFidelity-Beta-Interface-PR12935-087c6ff730afe126fb19e09da05b6d312db27972.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9574/app_12935_PR_9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9574/HighFidelity-Beta-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9574/HighFidelity-Beta-Interface-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9574/HighFidelity-Beta-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9574/HighFidelity-Beta-Interface-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.exe

**Comment by gcalero** [comment]
Build this please

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9707/HighFidelity-Beta-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9707/HighFidelity-Beta-Interface-PR12935-9b63b76b684aa86a456da330c8ac4deb9ca0c1f7.exe

**Comment by conklin94122** [comment]
When https://github.com/highfidelity/hifi/pull/12939 has passed qa and is merged, we need to rebuild this pr sop it picks up the changes in 939, then this pr is qa ready.

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9819/HighFidelity-Beta-Interface-PR12935-4a3001be1894545ca4c52fb278fe5bab34e7fef4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9819/HighFidelity-Beta-PR12935-4a3001be1894545ca4c52fb278fe5bab34e7fef4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9819/HighFidelity-Beta-Interface-PR12935-4a3001be1894545ca4c52fb278fe5bab34e7fef4.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9833/HighFidelity-Beta-PR12935-5afc9c6df33955a16148c4cb70865153ea61de17.apk

**Comment by gcalero** [comment]
Could you QA this please?

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9833/HighFidelity-Beta-PR12935-5afc9c6df33955a16148c4cb70865153ea61de17.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9833/HighFidelity-Beta-Interface-PR12935-5afc9c6df33955a16148c4cb70865153ea61de17.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9833/HighFidelity-Beta-PR12935-5afc9c6df33955a16148c4cb70865153ea61de17.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9833/HighFidelity-Beta-Interface-PR12935-5afc9c6df33955a16148c4cb70865153ea61de17.exe

File: android/app/src/main/java/io/highfidelity/hifiinterface/InterfaceActivity.java [hunk 1]
```diff
@@ -195,13 +188,16 @@ protected void onNewIntent(Intent intent) {
...
         }
-        nativeGoBackFromAndroidActivity();
     }
 
-    public void openAndroidActivity(String activityName) {
+    public void openAndroidActivity(String activityName, boolean backToScene) {
         switch (activityName) {
-            case "Home": {
-                Intent intent = new Intent(this, HomeActivity.class);
+            case "Home":
```
Comment (SamGondelman) [inline android/app/src/main/java/io/highfidelity/hifiinterface/InterfaceActivity.java]: We should be consistent with "Home" vs. "Main".  If the activity is MainActivity now, we should change all of this to "Main".

File: android/app/src/main/java/io/highfidelity/hifiinterface/fragment/LoginFragment.java [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+            public void onTextChanged(CharSequence charSequence, int start, int count, int after) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+                if (!ignoreNextChange) {
+                    ignoreNextChange = true;
+                    boolean spaceFound=false;
+                    for (int i=0; i< editable.length(); i++) {
```
Comment (SamGondelman) [inline android/app/src/main/java/io/highfidelity/hifiinterface/fragment/LoginFragment.java]: spacing (lines 75. 76. 79)

File: android/app/src/main/res/layout/fragment_home.xml [hunk 3]
```diff
@@ -6,8 +6,8 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     app:layout_behavior="@string/appbar_scrolling_view_behavior"
-    android:background="@color/colorPrimary"
-    tools:context="io.highfidelity.hifiinterface.HomeActivity"
+    android:background="@color/colorPrimary"
```
Comment (SamGondelman) [inline android/app/src/main/res/layout/fragment_home.xml]: extra spaces

File: android/app/src/main/java/io/highfidelity/hifiinterface/InterfaceActivity.java (hunk 1 above)
Comment (gcalero) [reply android/app/src/main/java/io/highfidelity/hifiinterface/InterfaceActivity.java]: Home as well as Login and other "Activities" are implemented now as fragment which allow us to reuse the menu and the action bar. This is why we open a MainActivity and pass "Home", "Login", etc as an extra parameter to it. Other (future) user activities that do not share the context (menu, action bar, etc.) may be implemented as real Activities. The caller to openAndroidActivity should not notice the difference nor know how it is implemented.

Comment (SamGondelman) [reply android/app/src/main/java/io/highfidelity/hifiinterface/InterfaceActivity.java]: Ah, I see.  In the future, how will someone calling openAndroidActivity distinguish between opening a new activity and opening a new fragment within an activity?  Maybe we should have two parameters, the first being activity ("Main") and the second being fragment ("Home", "Login", etc.).

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9850/HighFidelity-Beta-PR12935-5370cb7725b7f5427058b49e6aaf2bc64f036a27.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9850/HighFidelity-Beta-Interface-PR12935-5370cb7725b7f5427058b49e6aaf2bc64f036a27.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9850/HighFidelity-Beta-PR12935-5370cb7725b7f5427058b49e6aaf2bc64f036a27.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9850/HighFidelity-Beta-Interface-PR12935-5370cb7725b7f5427058b49e6aaf2bc64f036a27.exe

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9860/HighFidelity-Beta-PR12935-65a9b98b97f72112892b6d0f4102d38eb9b08b0e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9860/HighFidelity-Beta-Interface-PR12935-65a9b98b97f72112892b6d0f4102d38eb9b08b0e.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9860/HighFidelity-Beta-PR12935-65a9b98b97f72112892b6d0f4102d38eb9b08b0e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9860/HighFidelity-Beta-Interface-PR12935-65a9b98b97f72112892b6d0f4102d38eb9b08b0e.dmg

File: interface/src/AndroidHelper.cpp [hunk 4]
```diff
@@ -45,8 +45,8 @@ void AndroidHelper::notifyLoadComplete() {
     emit qtAppLoadComplete();
 }
 
-void AndroidHelper::performHapticFeedback(const QString& feedbackConstant) {
-    emit hapticFeedbackRequested(feedbackConstant);
+void AndroidHelper::notifyEnterForeground() {
+    emit enterForeground();
```
Comment (SamGondelman) [inline interface/src/AndroidHelper.cpp]: Why'd you change this?

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9879/HighFidelity-Beta-PR12935-5a73da6fdfa35efcecd86679b0d1e960aa8ceee5.apk

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9881/HighFidelity-Beta-PR12935-e10b71650766d0e423b0f0ebcfbaa88a6fa9f5e1.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9881/HighFidelity-Beta-PR12935-e10b71650766d0e423b0f0ebcfbaa88a6fa9f5e1.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9881/HighFidelity-Beta-Interface-PR12935-e10b71650766d0e423b0f0ebcfbaa88a6fa9f5e1.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9881/HighFidelity-Beta-PR12935-e10b71650766d0e423b0f0ebcfbaa88a6fa9f5e1.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9881/HighFidelity-Beta-Interface-PR12935-e10b71650766d0e423b0f0ebcfbaa88a6fa9f5e1.exe

**Comment by qlimguicheteau** [comment]
Currently Testing Android Interface

**Comment by qlimguicheteau** [comment]
Pass @conklin94122 
PR Test works as expected, Two phones and Desktop passed the smoke.

**Pull request merged**